src/vireshJJun20/Assignment26/SumOfNumbersInString.java

```diff
@@ -0,0 +1,28 @@
+package vireshJJun20.Assignment26;
+
+public class SumOfNumbersInString {
+
+	int sum = 0;
+	String tempStr = "";
+	void findSum(String str) {
+		for (int index = 0; index < str.length(); index++) {
+			if (Character.isDigit(str.charAt(index))) {
+				tempStr = tempStr + str.charAt(index);
```

Comment:
code looks good . at line 10 you can use 

tempStr += str.charAt(index);

Reply:
@DeavinaSJun20 thank you for comment.